.github/workflows/tests.yml

```diff
@@ -0,0 +1,112 @@
+---
+name: Tests
+
+on:
+  - push
+  - pull_request
```

Comment:
I think we want one or the other, but not both, otherwise we'll run this twice (see the checks on this PR). IMO, running on all push events makes sense, but we may want to wait for a PR, not sure.

Reply:
Done